src/app/issuer/components/badgeclass-detail/badgeclass-detail.component.ts

```diff
@@ -226,12 +224,12 @@ export class BadgeClassDetailComponent extends BaseAuthenticatedRoutableComponen
 							},
 							(error) => {
 								this.messageService.reportAndThrowError(
-									`Failed to delete badge class: ${BadgrApiFailure.from(error).firstMessage}`
+									`Failed to delete badge class: ${BadgrApiFailure.from(error).firstMessage}`,
```

Comment:
And so on, I won't add a comment to every single occurrence. But why did you add commas everywhere?